nexus.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+var fluid = require("infusion"),
+    kettle = fluid.registerNamespace("kettle"),
+    path = require("path");
+
+require("kettle");
```

Comment:
This base driver should contain a fluid.module.register directive to allow module-based path interpolation, for example as with https://github.com/GPII/gpii-express/blob/master/index.js#L14